docs/developer/images.md

```diff
@@ -0,0 +1,128 @@
+# Art and Images
+
+## Colors
+
+Images in arcade are stored using 4 bits per color. The color `0` is
+reserved for transparency, so there are 15 available colors for you to
+work with at any one time. Colors are represented as indices into a palette;
+the default palette shown in the MakeCode UI is here:
+
+```
+#000000     // transparency
+#ffffff
+#ff2121
+#ff93c4
+#ff8135
+#fff609
+#249ca3
+#78dc52
+#003fad
+#87f2ff
+#8e2ec4
+#a4839f
+#5c406c
+#e5cdc4
+#91463d
+#000000
+```
+
+
+### Changing the default palette
+
+If you intend to use the MakeCode blocks editor for
```

Comment:
If you intend to use the MakeCode blocks editor to create or edit images for... --- I'm guessing this is what is meant